en/tutorials/tutorial_hello_sky.md

```diff
@@ -199,70 +256,70 @@ The `sensor_sub_fd` is a topic handle and can be used to very efficiently perfor
 
 Adding `poll()` to the subscription looks like (*pseudocode, look for the full implementation below*):
 
-```C++
+```cpp
 #include <poll.h>
 #include <uORB/topics/sensor_combined.h>
 ..
 int sensor_sub_fd = orb_subscribe(ORB_ID(sensor_combined));
 
 /* one could wait for multiple topics with this technique, just using one here */
 px4_pollfd_struct_t fds[] = {
-	{ .fd = sensor_sub_fd,   .events = POLLIN },
+    { .fd = sensor_sub_fd,   .events = POLLIN },
 };
 
 while (true) {
-	/* wait for sensor update of 1 file descriptor for 1000 ms (1 second) */
-	int poll_ret = px4_poll(fds, 1, 1000);
+uORB/* wait for sensor update of 1 file descriptor for 1000 ms (1 second) */
+uORBint poll_ret = px4_poll(fds, 1, 1000);
 ..
-	if (fds[0].revents & POLLIN) {
-		/* obtained data for the first file descriptor */
-		struct sensor_combined_s raw;
-		/* copy sensors raw data into local buffer */
-		orb_copy(ORB_ID(sensor_combined), sensor_sub_fd, &raw);
-		PX4_INFO("Accelerometer:\t%8.4f\t%8.4f\t%8.4f",
-					(double)raw.accelerometer_m_s2[0],
-					(double)raw.accelerometer_m_s2[1],
-					(double)raw.accelerometer_m_s2[2]);
-	}
+    if (fds[0].revents & POLLIN) {
+        /* obtained data for the first file descriptor */
+        struct sensor_combined_s raw;
+        /* copy sensors raw data into local buffer */
+        orb_copy(ORB_ID(sensor_combined), sensor_sub_fd, &raw);
+        PX4_INFO("Accelerometer:\t%8.4f\t%8.4f\t%8.4f",
+                    (double)raw.accelerometer_m_s2[0],
+                    (double)raw.accelerometer_m_s2[1],
+                    (double)raw.accelerometer_m_s2[2]);
+    }
 }
 ```
 
-Compile the app now by issuing
+Compile the app again by entering:
 
-<div class="host-code"></div>
-
-```
-  make
+```sh
```

Comment:
@bkueng Does calling make after doing a previous make cache the type of make request?

Reply:
No